packages/eds-core-react/src/components/Slider/Slider.stories.tsx

```diff
@@ -1,29 +1,67 @@
 import { useState, FormEvent } from 'react'
 import { Slider, SliderProps } from '../..'
-import { Story, Meta } from '@storybook/react'
+import { Story, ComponentMeta } from '@storybook/react'
+import page from './Slider.docs.mdx'
 
 export default {
   title: 'Inputs/Slider',
   component: Slider,
   parameters: {
     docs: {
-      description: {
-        component: `A slider is a flexible input that allows users to adjust
-        predefined values.
-        `,
-      },
+      page,
     },
   },
-} as Meta
+} as ComponentMeta<typeof Slider>
 
-export const Default: Story<SliderProps> = (args) => {
+export const Introduction: Story<SliderProps> = (args) => {
   return (
     <div style={{ margin: '3rem' }}>
       <Slider {...args} />
     </div>
   )
 }
 
+export const SimpleSlider: Story<SliderProps> = () => (
+  <>
+    <span id="simple-slider">Slide me</span>
+    <Slider value={4} min={0} max={10} ariaLabelledby="simple-slider" />
+  </>
+)
+SimpleSlider.storyName = 'Simple slider'
+SimpleSlider.decorators = [
```

Comment:
Suggest using `<Stack>` component from our Storybook components instead of this.

Reply:
I would have to overwrite display:
`const Stack = styled(SBStack)
  margin: 3rem;
  display: block;
`
Would it be right to do so @mimarz ?

Reply:
Can't you just use `Stack` as it is?

Reply:
Will look like that, will change span position:
<img width="1016" alt="Screenshot 2022-04-20 at 11 46 29" src="https://user-images.githubusercontent.com/7075271/164200860-94568ef3-52e7-4e92-9580-c82afc0ba6bd.png">

Reply:
Ah, this can be solved by changing the `flex-direction` to `row`?

Reply:
> I bet there are other use-case we want different direction in our stories.

Yes indeed

Reply:
What about the `gap` between the elements? Also elements are align `centre` this will move `span` text into the centre? 🤔

Reply:
The more I look at it the more I feel we need to change `Slider`'s props. I don't quite understand why `ariaLabelledby` is mandatory as using only `aria-label` is also valid a11y...  Reference: https://www.w3.org/TR/wai-aria-practices/#slider_roles_states_props

Ok, so maybe just leave it and then we can change it after we have fixed the properties? I'll make a follow up issue.

Reply:
I see, ok i will do so then.

Reply:
Made an issue regarding this and some initial thoughts #2167 😅